doc/source/whatsnew/v1.2.0.rst

```diff
@@ -475,6 +475,7 @@ Indexing
 - Bug in :meth:`Series.loc` and :meth:`DataFrame.loc` raises when numeric label was given for object :class:`Index` although label was in :class:`Index` (:issue:`26491`)
 - Bug in :meth:`DataFrame.loc` returned requested key plus missing values when ``loc`` was applied to single level from :class:`MultiIndex` (:issue:`27104`)
 - Bug in indexing on a :class:`Series` or :class:`DataFrame` with a :class:`CategoricalIndex` using a listlike indexer containing NA values (:issue:`37722`)
+- Bug in :meth:`DataFrame.loc` raised ``TypeError`` when non-integer :class:`Interval` was given to select values from :class:`MultiIndex` (:issue:`25165`, :issue:`24263`)
```

Comment:
the tests dont seem to have anything to do with Intervals.  Am i missing something?

Reply:
No, you are right. Poor wording on my side. I meant a slice.